app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,44 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :photo %><br />
+    <%= f.text_field :photo, autofocus: true, autocomplete: "photo" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :bio %><br />
+    <%= f.text_area :bio, autofocus: true, autocomplete: "bio" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
```

Comment:
- [ ] This is the view page for registering a new User. You provided a name field, but it did not appear on the screen. I am not sure what is wrong. Kindly check to ensure everything is okay. 

![image](https://user-images.githubusercontent.com/84156714/231317439-ee020c8e-72b3-4d18-adda-2f8343da9120.png)

Reply:
![Captura desde 2023-04-11 19-24-19](https://user-images.githubusercontent.com/113067895/231323383-f30aa51f-985e-4976-9f7e-128350ab593b.png)
I have changed the styles, but before that changes the field name was in the page